package.json

```diff
@@ -16,6 +16,7 @@
     "checkTs": "tsc --noEmit",
     "lint": "eslint --ext js,ts,tsx ./",
     "lint:fix": "yarn run lint --fix",
+    "gitlint": "node ./internals/scripts/gitlint.js",
```

Comment:
(should we rewrite it in python, so it seems less like a nodejs package?)

Reply:
He he, I think there's no need for that :slightly_smiling_face: